### PR TITLE
[idea][bug 475118] install Xtend library only when a project is initialized

### PR DESCRIPTION
**Pull request by akosyakov**
Change-Id: I44e785d35f15dd4c729a614412f16537451bae80
Signed-off-by: akosyakov <anton.kosyakov@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=475118

**Comment by akosyakov** [comment]
Jenkins intellij-tests-master-latest

File: intellij/org.eclipse.xtend.idea/src/org/eclipse/xtend/core/idea/config/XtendSupportConfigurable.xtend [hunk 1]
```diff
@@ -71,7 +71,7 @@ class XtendSupportConfigurable extends FrameworkSupportInModuleConfigurable {
 		val parent = sibling?.file?.parent
 		if (parent === null)
 			return null
-		val existing = parent.children.findFirst[it.name == name]
```
Comment (szarnekow) [inline intellij/org.eclipse.xtend.idea/src/org/eclipse/xtend/core/idea/config/XtendSupportConfigurable.xtend]: can we use com.intellij.openapi.vfs.VfsUtil.createDirectoryIfMissing(VirtualFile, String) ?

**Comment by akosyakov** [comment]
Jenkins intellij-tests-master-latest

**Comment by akosyakov** [comment]
Jenkins test

**Comment by akosyakov** [comment]
Jenkins intellij-tests-master-latest

**Comment by akosyakov** [comment]
Jenkins intellij-tests-master-latest

File: intellij/org.eclipse.xtend.idea/src/org/eclipse/xtend/core/idea/config/XtendLibraryManager.xtend [hunk 2]
```diff
@@ -37,7 +38,8 @@ import org.jetbrains.plugins.gradle.util.GradleConstants
 import org.jetbrains.plugins.groovy.lang.psi.GroovyPsiElementFactory
 import org.jetbrains.plugins.groovy.lang.psi.api.statements.expressions.GrCall
 import org.jetbrains.plugins.groovy.lang.psi.api.statements.expressions.GrMethodCall
-import org.jetbrains.annotations.NotNull
+
+import static org.eclipse.xtend.core.idea.config.XtendLibraryManager.*
```
Comment (dhuebner) [inline intellij/org.eclipse.xtend.idea/src/org/eclipse/xtend/core/idea/config/XtendLibraryManager.xtend]: self static import already removed in master. Rebase?

**Pull request merged**